File: 파이썬3/프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    n1 = 0; n2 = 0
    for i in range(len(num_list)):
        if i%2==0:
            n2+=num_list[i]
        else:
            n1+=num_list[i]
    return max(n1,n2)

File: C#/프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int n1=0,n2=0;
        for (int i=0;i<num_list.Length;i++){
            if (i%2==0){
                n2+=num_list[i];
            }
            else{
                n1+=num_list[i];
            }
        }
        return Math.Max(n1,n2);
    }
}